core/src/components/range/range.tsx

```diff
@@ -142,6 +143,29 @@ export class Range implements ComponentInterface {
    */
   @Prop() ticks = true;
 
+  /**
+   * The start position of the range active bar, only available with a single knob (dualKnobs="false").
+   * Valid values are between the `min` and `max` values.
```

Comment:
```suggestion
   * Valid values are between the `min` and `max` values.
```

Not quite sure how best to word this, but it would be nice to state that it is inclusive of `max` and `min`.